charm-slurmctld/actions.yaml

```diff
@@ -45,3 +45,22 @@ influxdb-info:
 
     This action returns the host, port, username, password, database, and
     retention policy regarding to InfluxDB.
+
+etcd-get-root-password:
+  description: >
+    Get the password for the etcd root account.
+
+etcd-create-munge-account:
+  description: >
+    Create a new etcd account to be able to query the munge key.
+
+  params:
+    user:
```

Comment:
This is non-consequential really, but thought I should bring it up anyway..... 

Does it really matter what the username is here? Possibly we can be opinionated about the username and choose something up front. This way the user wouldn't have to decide and configure this additional param. Thoughts?

Reply:
I see below ... it almost makes it easier to maintain the code if we just treat them equally and just get and set them both the same way. Ok, disregard my comment above.... if there is no impact on the end user, we should always choose what is most maintainable.